reptiles.html

```diff
@@ -42,20 +46,72 @@
           </label>
           <div class="sidebar">
             <nav class="sidebar-menu">
-              <ul>
-                <!--<li>
```

Comment:
Your sidebar goes over the header when you scroll down. Others have put the header below the main content.
Alex:
<img width="390" alt="Screenshot 2024-12-09 at 12 33 13" src="https://github.com/user-attachments/assets/ceec2ddc-71a6-47ed-bf6a-a2ea5d97ddbe">
yours:
<img width="392" alt="Screenshot 2024-12-09 at 12 33 29" src="https://github.com/user-attachments/assets/afc08e61-9f1b-43cd-aaec-89cd0d4aab20">